sw/device/lib/arch/device.h

```diff
@@ -29,22 +29,22 @@ typedef enum device_type {
   /**
    * Represents "DV", i.e. running th test in a DV simulation testbench.
    */
-  kDeviceSimDV,
+  kDeviceSimDV = 0,
```

Comment:
i actually didn't realize this device arch stuff was linked into test rom build, this is pretty cool.